drivers/hmis/spec/requests/hmis/lookup_client_spec.rb

```diff
@@ -147,6 +154,7 @@
       'phoneNumbers' => [{ 'value' => '5554567891', 'use' => 'home', 'system' => 'phone' }],
       'emailAddresses' => [{ 'value' => 'email@e.mail', 'use' => 'home', 'system' => 'email' }],
     )
+    expect(result.dig('data', 'client', 'image')).not_to be_nil
   end
 
```

Comment:
Can you add a test where it removes `can_view_client_photo` and ensures the photo is not resolved?

Reply:
Ah, yeah of course! Sorry I forgot this.